CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [3.1.0] - 2017-11-07
```

Comment:
as there is no guarantee when we get to PRs or the order that they are merged we accept contributions as [Unreleased change](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#unreleased-change) and leave the versioning to the maintainers this allows us to also validate that the contribution is versioned correctly from a semver perspective.